portal/helpers/emails.py

```diff
@@ -130,6 +130,36 @@ def send_verification_email(request, user, new_email=None):
                    message['message'])
 
 
+def send_verification_email_new(request, user, new_email=None):
```

Comment:
likewise, I'd rather you stay consistent with your redesign process and copy the file as _new.py :)